ext_autoload.php

```diff
@@ -1,71 +1,198 @@
 <?php
-// generate automatically by extdeveval::buildAutoloadRegistry.
-/*
- * Seit Version 4.3 bietet TYPO3 einen Autoloader der sehr einfach einzusetzen ist.
- * In dieser Datei machen wir nichts weiteres als ein assoziatives Array zurückzugeben.
- *
- * Die grösste Fehlerquelle dabei ist, dass man darauf achten muss,
- * den Schlüssel des assoziativen Arrays klein zu schreiben.
- * Verwendet man den korrekten Klassennamen, also camel case,
- * wird die Klasse nicht eingebunden.
- *
- * Um die Datei automatisch generieren zu lassen, kann die Extension extdeveval genutzt werden.
- */
+// DO NOT CHANGE THIS FILE! It is automatically generated by extdeveval::buildAutoloadRegistry.
+// This file was generated on 2016-07-01 20:06
 
-// wir können weder den Extensionmanager nutzen (auf Grund der verschiedenen TYPO3 Versionen)
-// noch tx_rnbase_util_Extensions (weil noch nicht geladen). Also nehmen wir den Pfad hart. Dürfte
-// aber kein Problem sein.
-$extensionPath = PATH_typo3conf . 'ext/mklib/';
```

Comment:
Please keep this line as it is. We don't want to use the extension manager directly at this point to be independent of the TYPO3 version. Or change it to the following:
```
if (class_exists('TYPO3\\CMS\\Core\\Utility\\ExtensionManagementUtility', true)) {
	$extensionPath = \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath('mklib');
} else {
	$extensionPath = t3lib_extMgm::extPath('mklib');
}
```
The rn_base wrapper can't be used as it might not be loaded at that point in older TYPO3 versions.